index.html

```diff
@@ -194,49 +198,228 @@ <h2>
+                    <a>rejected</a> with a newly
+                    <a data-link-for="exception" data-lt="create">created</a>
+                    <code>TypeError</code>.</p>
+                  </li>
+                </ol>
+              </li>
+              <li>
+                <p>Let <var>requestedMediaTypes</var> be the set of media
+                types in <var>constraints</var> with either a dictionary
+                value or a value of <code>true</code>.</p>
+              </li>
+              <li>
+                <p>If <var>requestedMediaTypes</var> is the empty set, set
+                <var>requestedMediaTypes</var> to a set containing
+                <code>"video"</code>.</p>
+              </li>
+              <li>
+                <p>If the <a>current settings object</a>'s <a>responsible
+                  document</a> is NOT <a data-cite=
+                    "!HTML52/browsers.html#fully-active">fully active</a>, return
+                  a promise <a>rejected</a> with a
+                  <code><a>DOMException</a></code> object whose
+                  <code><a>name</a></code> attribute has the value
+                  <code>InvalidStateError</code>.</p>
+              </li>
+              <li>
+                <p>If the <a>current settings object</a>'s <a>responsible
+                  document</a> is NOT <a>allowed to use</a> the feature
+                indicated by Feature Policy [TBD],
+                return a promise <a>rejected</a> with a
+                <code><a>DOMException</a></code> object whose
+                <code><a>name</a></code> attribute has the value
+                <code>SecurityError</code>.</p>
+              </li>
+              <li>
+                <p>Let <var>originIdentifier</var> be the <a>current settings
+                  object</a>'s <a data-cite=
+                    "!HTML52/webappapis.html#responsible-browsing-context">responsible
+                    browsing context</a>'s [[!HTML52]] <a data-cite=
+                      "!HTML52/browsers.html#top-level-browsing-context">top-level
+                      browsing context</a>'s <a data-cite=
+                        "!HTML52/browsers.html#active-document">active document</a>'s
+                      origin.</p>
+              </li>
+              <li>
+                <p>If the <a>current settings object</a>'s origin is
+                different from <var>originIdentifier</var>, set
+                <var>originIdentifier</var> to the result of combining
+                <var>originIdentifier</var> and the <a>current settings
+                  object</a>'s origin.</p>
+              </li>
+              <li>
+                <p>Let <var>p</var> be a new promise.</p>
+              </li>
+              <li>
+                <p>Run the following steps in parallel:</p>
+                <ol>
+                  <li>
+                    <p>For each media type <var>T</var> in
```

Comment:
Is "media type" well defined? Should it be <a></a> or a link, or somewhere clarified that this is "audio" or "video"?